lib/friendly_id/base.rb

```diff
@@ -190,6 +190,7 @@ module Base
     # @yieldparam config The model class's {FriendlyId::Configuration friendly_id_config}.
     def friendly_id(base = nil, options = {}, &block)
       yield friendly_id_config if block_given?
+      friendly_id_config.options = options
```

Comment:
As for me, it's better to follow existing concept of explicitly specified configuration options and not to add whole hash. By the way, this config is available only for `history` module. So, we should clarify this fact in method's documentation above.